internal/output/table.go

```diff
@@ -75,5 +79,14 @@ func PrintTableResults(vulnResult *models.VulnerabilityResults, outputWriter io.
 	if outputTable.Length() == 0 {
 		return
 	}
+
 	outputTable.Render()
 }
+
+func wrapPath(path string) string {
+	slashPath := filepath.ToSlash(path)
+	moddedPath := strings.ReplaceAll(slashPath, "/", "/ ")
+	wrappedPath := wordwrap.WrapString(moddedPath, sourceLength)
+	unmoddedWrappedPath := strings.ReplaceAll(wrappedPath, "/ ", "/")
```

Comment:
Could there be a source path with a space as the first character that would break this? 

This seems a bit too hacky to rely on. I'd rather we just write our own wrapping library for this case.